common/artifacts/909_wc_artifacts_library_librams.txt

```diff
@@ -110,14 +106,12 @@ libram_of_compassion = {
 	indestructible = yes
 
 	diplomacy = 2
-	learning = 2
-	combat_rating = 5
+	
+	global_revolt_risk = -0.03
 
 	general_opinion = 5
 	light_group_opinion = 5
 	
-	monthly_character_prestige = 0.2
-	monthly_character_piety = 0.4
```

Comment:
please keep to 2xtier cost from prestige/piety gain
legendary libram without prestie/piety? I don't think so